lxd/main_forkconsole.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"os"
+	"strconv"
+	"strings"
+
+	"gopkg.in/lxc/go-lxc.v2"
+)
+
+/*
+ * This is called by lxd when called as "lxd forkconsole <container> <path> <conf> <tty=<n>> <escape=<n>>"
+ */
+func cmdForkConsole(args *Args) error {
```

Comment:
Needs to be added to the help in main_args.go

Reply:
done